src/Http/Controllers/SessionController.php

```diff
@@ -80,4 +117,13 @@ protected function generateAuthToken($user) {
 
         return $jwt->encode();
     }
+
+    protected function generateTokenAndSendResponse($user) {
+        if ($user) {
+            $token = $this->generateAuthToken($user);
+            return response()->json(['token' => $token]);
+        }
+        Log::info("401");
```

Comment:
I guess it will crash as the `Log` is no longer "required". Take the time to check if there is remaining call in the folder when you remove a "required" component.